src/nni_manager/core/sqlDatabase.ts

```diff
@@ -178,17 +178,21 @@ class SqlDB implements Database {
     public queryTrialJobEvent(trialJobId?: string, event?: TrialJobEvent): Promise<TrialJobEventRecord[]> {
         let sql: string = '';
         let args: any[] | undefined;
+        let jobId: string | undefined;
+        if (trialJobId && trialJobId.search("-") > 0) {
```

Comment:
`trialJobId` is in "jobid-parameterid" format?

Reply:
yes